manifests/config.pp

```diff
@@ -15,6 +15,7 @@
     group   => $katello::group,
     mode    => '0644',
     before  => [Class['foreman::database'], Exec['foreman-rake-db:migrate']],
+    notify  => [Service['foreman-tasks'], Service['httpd']],
```

Comment:
I thought this was the same as https://github.com/Katello/puppet-katello/pull/62, but also here you may want to notify ```Class['foreman::service']```.